client/jobs.lua

```diff
@@ -81,46 +78,56 @@ local function startWork()
     StopAnimTask(cache.ped, "anim@gangops@facility@servers@", "hotwire", 1.0)
 end
 
-local function canInteractWithBox(i)
-    return InJail and CurrentJob and not isWorking and i == currentBox
+local function canInteractWithTask(i)
+    return InJail and prisonJob and not isWorking and i == currentTask
+end
+
+function chooseJob(job)
+    prisonJob = job
+    jobLocations = newRandomArray(#Config.Jobs[prisonJob].locations)
 end
 
--- Threads
+exports('getPrisonJob', function()
+    return prisonJob
+end)
+
 CreateThread(function()
-    for i = 1, #Config.Jobs.electrician.locations do
-        local coords = Config.Jobs.electrician.locations[i]
-        if Config.UseTarget then
-            exports.ox_target:addBoxZone({
-                coords = coords,
-                size = vec3(1.5, 1.6, 5),
-                options = {
-                    {
-                        icon = 'fa-solid fa-bolt',
-                        label = locale("info.job_interaction_target", "Electrician"),
-                        canInteract = function()
-                            return canInteractWithBox(i)
-                        end,
-                        onSelect = startWork
+    if prisonJob then
+        for i = 1, #Config.Jobs[prisonJob].locations do
+            local coords = Config.Jobs[prisonJob].locations[i]
+            if Config.UseTarget then
+                exports.ox_target:addBoxZone({
+                    coords = coords,
+                    size = vec3(1.5, 1.6, 5),
+                    options = {
+                        {
+                            icon = 'fa-solid fa-bolt',
+                            label = locale("info.job_interaction_target", prisonJob),
```

Comment:
```suggestion
                            label = locale("info.job_interaction_target", prisonJob.label),
```
Does this locale need to change as well?